airbyte-integrations/connectors/destination-bigquery/src/test-integration/java/io/airbyte/integrations/destination/bigquery/BigQueryIntegrationTest.java

```diff
@@ -120,10 +120,13 @@ protected boolean implementsBasicNormalization() {
   }
 
   private List<JsonNode> retrieveRecordsFromTable(TestDestinationEnv env, String tableName) throws InterruptedException {
+    // todo (cgardens) - see https://github.com/airbytehq/airbyte/pull/1446.
+    final String emittedAtColumn =
+        tableName.toLowerCase().endsWith("raw") || tableName.toLowerCase().endsWith("raw\"") ? "emitted_at" : "_airbyte_emitted_at";
```

Comment:
why do we have the `||`?

Reply:
Shouldn't we have `_airbyte` all the time? why the distinction between raw and not raw?

Reply:
depending on the db and the test sometimes it uses quoted syntax. sometimes not.